Tuturial-Structure-Template.md

```diff
@@ -1,68 +1,178 @@
+> 
+> **IMPORTANT**: The first prerequisite will always include a link to the _“How to get started with Agora”_ tutorial using URL: https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=
+> 
+> For example:
 
-The Build XYZ section should be broken down into subsections that build the project in smaller chunks. This section could explain how to implement a set of functions that operate together to produce a specific feature of the project. Be sure to explain all inputs and outputs, discuss their significance within the code and how they work together.  
 
-Within the Build and Build Subsections there should be code snippets included. All snippets should use GitHub Gists _(on Agora Blog and Medium)_ and ``` within Dev.to or .md files.  
+To successfully understand and complete this task you must have:
 
-When embedding code, be sure to include the language name to ensure the proper code highlighting when the markdown renders. For example, for javascript code snippets use: ```javascript  
+- A valid [Agora account](https://docs.agora.io/en/Agora%20Platform/sign_in_and_sign_up). If you don't have an account, see how to [Get started with Agoras](https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=PAGE_TITLE).
+  > The URL to the setup the Agora account is made up of _URL_:_PAGE_TITLE_. 
+  > For a blog named _How to Build a Token Server for Agora Applications using GoLang_ the link is: https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=How_to_Build_a_Token_Server_for_Agora_Applications_using_GoLang
+  > 
+  > It's imperative that you type this URL correctly as it will be the primary way Agora can properly track developer conversion from individual content posts.
+- An Agora project with the [app certificate](https://docs.agora.io/en/Agora%20Platform/manage_projects?platform=All%20Platforms#manage-your-app-certificates) enabled.  
+- Good knowledge of [Multi-factor authentication](https://en.wikipedia.org/wiki/Multi-factor_authentication).
 
-When embedding code using GitHub Gists, if the code is only a snippet make sure to denote that in the file name following the format: `functionName+snippet.fileExtension`  
 
-For example if my function name is generateToken and its a javascript snippet, the Gist would have the title: `generateToken+snippet.js`  
+## Project Setup 
 
-If the code within the Gist is an entire file or set of functions that can be copied and pasted and will work on their own as a library or project, you can just name the Gist after the file they would represent with the repository.  
+> Describe the steps to create a new project and set up the environment.
```

Comment:
A few things that I think can go to the project setup chapter:
* Create a xxx project
* Necessary configurations
* Create a UI
* Import the SDK

I am not entirely sure whether import the SDK fits

Reply:
I think the SDK import should be a separate step, as project setup may be optional for some developers.

Reply:
This is for DevRel only, we don't mind what it is. 

Disagree about the SDK import. Our target platform is npm so doing the npm import is before coding. Same thing for Android etc.